docs/core/diagnostics/diagnostic-port.md

```diff
@@ -0,0 +1,61 @@
+
+## Security Considerations
+
+The diagnostic port exposes sensitive information about a running application. If an untrusted user gains access to this channel they could observe detailed program state, including any secrets which are in memory, and arbitrarily modify the execution of the program. On the CoreCLR runtime the default diagnostic port is configured to only be accessible by the same user account which launched the app or by an account with super-user permissions. If your security model does not trust other processes with the same user account credentials all diagnostic ports can be disabled by setting environment variable `DOTNET_EnableDiagnostics=0`. Doing this will block your ability to use external tooling such as .NET debugging or any of the dotnet-* diagnostic tools.
+
+[!INCLUDE [complus-prefix](../../../includes/complus-prefix.md)]
+
+## Default Diagnostic Port
+
+On Windows, Linux and MacOS the runtime has one diagnostic port open by default at a well-known endpoint. This is the port that the dotnet-* diagnostic tools are connecting to automatically when they haven't been explicitly configured to use an alternate port. The endpoint is:
+
+- Windows - Named Pipe `\\.\pipe\dotnet-diagnostic-{pid}`
+- Linux and MacOS - Unix Domain Socket `{temp}/dotnet-diagnostic-{pid}-{disambiguation_key}-socket`
+
+`{pid}` is the process id written in decimal, `{temp}` is the `TMPDIR` environment variable or the value `/tmp` if `TMPDIR` is undefined/empty, and `{disambiguation_key}` is the process start time written in decimal. On MacOS and NetBSD the process start time is number of seconds since UNIX epoch time and on all other platforms it is jiffies since boot time.
```

Comment:
```suggestion
`{pid}` is the process id written in decimal, `{temp}` is the `TMPDIR` environment variable or the value `/tmp` if `TMPDIR` is undefined/empty, and `{disambiguation_key}` is the process start time written in decimal. On macOS and NetBSD, the process start time is number of seconds since UNIX epoch time and on all other platforms it is jiffies since boot time.
```